sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -100,6 +101,73 @@ public async Task VerifyImdsRequestWithClientIdMockAsync()
             Assert.AreEqual("true", metadataValue);
         }
 
+        [NonParallelizable]
+        [Test]
```

Comment:
Tip: when using `[TestCase]` or similar attributes, you don't *need* `[Test]`. Doesn't hurt, though. Just a tip.